website/content/docs/concepts/policies.mdx

```diff
@@ -800,32 +800,36 @@ authenticated user.
-| --------------------------------------------------------------------------------------------------------------------------------------------------------------------------- | ----------------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
-| [auth/token/accessors](/vault/api-docs/auth/token#list-accessors)                                                                                | LIST              | List token accessor                                                                                                                                 |
-| [auth/token/create-orphan](/vault/api-docs/auth/token#create-token)                                                                              | POST              | Create an orphan token (the same as `no_parent` option)                                                                                             |
-| [auth/token](/vault/api-docs/auth/token#create-token)                                                                                            | POST              | Create a periodic or an orphan token (`period` or `no_parent`) option                                                                               |
-| [pki/root](/vault/api-docs/secret/pki#delete-root)                                                                                               | DELETE            | Delete the current CA key ([pki secrets engine](/vault/docs/secrets/pki))                                           |
-| [pki/root/sign-self-issued](/vault/api-docs/secret/pki#sign-self-issued)                                                                         | POST              | Use the configured CA certificate to sign a self-issued certificate ([pki secrets engine](/vault/docs/secrets/pki)) |
+| Path                                                                                                                                                   | HTTP verb         | Description                                                                                                                                         |
+| -------------------------------------------------------------------------------------------------------------------------------------------------------| ----------------- | --------------------------------------------------------------------------------------------------------------------------------------------------- |
+| [auth/token/accessors](/vault/api-docs/auth/token#list-accessors)                                                                                      | LIST              | List token accessor                                                                                                                                 |
```

Comment:
The description could be a bit more useful here - how about `List token accessors for all current Vault service tokens` ?

Reply:
Added to upcoming commit.